.env.development

```diff
@@ -1,7 +1,3 @@
-AUTH0_DOMAIN=dev-sbac.auth0.com
-AUTH0_CLIENT=rgZL4I04pep3P2GRIEVQtDkWcHjv9sru
-AUDIENCE=sendhug
-LOGIN_REDIRECT=http://localhost:3000/user
-LOGOUT_REDIRECT=http://localhost:3000
+BASE_URL=http://localhost:3000
 BACKEND_URL=http://127.0.0.1:5000
```

Comment:
any reason not to use localhost for this as well?

Reply:
At some point something changed (don't know if it's flask or macOS) and for some reason localhost isn't recognised anymore even though they're the same address... not entirely sure why